lib/ts/recipe/session/types.ts

```diff
@@ -256,7 +256,11 @@ export type RecipeInterface = {
     revokeMultipleSessions(input: { sessionHandles: string[]; userContext: any }): Promise<string[]>;
 
     // Returns false if the sessionHandle does not exist
```

Comment:
the input to createNewSession still uses `sessionData`. Please rename that to `sessionDataInDatabase` as well. Is there any other place where sessionData is being used?